src/Enable.Extensions.Queuing.AzureServiceBus/Internal/AzureServiceBusQueueClient.cs

```diff
@@ -119,7 +119,7 @@ public class AzureServiceBusQueueClient : BaseQueueClient
             {
                 AutoComplete = messageHandlerOptions.AutoComplete,
                 MaxConcurrentCalls = messageHandlerOptions.MaxConcurrentCalls,
-                MaxAutoRenewDuration = TimeSpan.FromMinutes(5)
+                MaxAutoRenewDuration = TimeSpan.FromMinutes(60)
```

Comment:
Would this be clearer if we used `TimeSpan.FromHours(1)`?

Reply:
Personally, not really any clearer. If it were 120 or 240 minutes it may be. Happy to make the change if you deem it so.

Reply:
Changed.